### PR TITLE
Update documentation.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,22 +7,24 @@
...
-- Highly scalable. Google's internal instance runs on over 25,000 machines.
+- Highly scalable. Can run on any size cluster (e.g. Google's instance runs on
+  30,000 VMs).
 - Accurate deduplication of crashes.
-- Fully automatic bug filing and closing for issue trackers
-  ([Monorail] only for now).
+- Fully automatic bug filing, triage and closing for various issue trackers
+  (e.g. [Monorail], [Jira]).
+- Support for coverage guided fuzzing (e.g. [libFuzzer], [AFL] and [Honggfuzz]).
+  Ensembles fuzzers for optimal fuzzing efficiency.
```
Comment (jonathanmetzman) [inline README.md:21]: I would say: "Supports multiple engines (e.g. Honggfuzz, AFL, LibFuzzer) for optimal results (ensemble fuzzing)".
The term "ensemble fuzing" isn't understandable to people who haven't read the paper.

**Review by jonathanmetzman — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (inferno-chromium) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -7,22 +7,25 @@
...
+  30,000 VMs).
 - Accurate deduplication of crashes.
-- Fully automatic bug filing and closing for issue trackers
-  ([Monorail] only for now).
+- Fully automatic bug filing, triage and closing for various issue trackers
+  (e.g. [Monorail], [Jira]).
+- Supports multiple coverage-guided fuzzing engines
+  ([libFuzzer], [AFL] and [Honggfuzz])
+  for optimal results (with [ensemble fuzzing] and [fuzzing strategies]).
+- Support for blackbox fuzzing.
```
Comment (oliverchang) [inline README.md:23]: nit: link to https://google.github.io/clusterfuzz/reference/coverage-guided-vs-blackbox/ ?

**Review by oliverchang — COMMENTED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (inferno-chromium) [reply README.md:23]: yes missed that, already fixed with https://github.com/google/clusterfuzz/pull/2040

**Pull request merged**